### ISSUE TITLE
Plotting/logging rewards

### ISSUE THREAD

**Issue by MarcoMeter** [PyTorchRL/pytorchrl#1]
Hi,

I worked through your [getting started tutorial](https://pytorchrl.readthedocs.io/en/latest/usage/getting_started.html#checking-results) to add the gym-minigrid environment. I think I've got the most important steps setup, while dealing with a few issues along the way. However, one important thing is missing. Any information on the achieved rewards is missing. Could you please provide a hint on how to add the mean reward to the console prints and the tensorboard?

I believe it has something to do with the key word argument `info_keywords`, but that seems to be individual for each environment.

Or is that the "performance/train".

The plotted losses look weird as well.

![plot](https://user-images.githubusercontent.com/6996955/105016259-b66efb00-5a42-11eb-870e-d6d478d8f18d.png)

```
import ray
import sys
import time

from pytorchrl import utils
from pytorchrl import Learner
from pytorchrl.scheme import Scheme
from pytorchrl.agent.algos import PPO
from pytorchrl.agent.env import VecEnv
from pytorchrl.agent.storages import GAEBuffer
from pytorchrl.agent.actors import OnPolicyActor, get_feature_extractor
from pytorchrl.envs import minigrid_train_env_factory

def main():
    ray.init()

    # 1. Define Train Vector of Envs
    train_envs_factory, action_space, obs_space = VecEnv.create_factory(
        vec_env_size=16,
        log_dir="/tmp/minigrid",
        env_fn=minigrid_train_env_factory,
        env_kwargs={"env_id": "MiniGrid-MemoryS9-v0"},
        info_keywords=("reward", "length"))

    # 3. Define RL training algorithm
    algo_factory = PPO.create_factory(
        lr=1e-4, num_epochs=4, clip_param=0.2, entropy_coef=0.01,
        value_loss_coef=.25, max_grad_norm=.5, num_mini_batch=8,
        use_clipped_value_loss=True, gamma=0.99)

    # 4. Define RL Policy
    actor_factory = OnPolicyActor.create_factory(
        obs_space, action_space,
        feature_extractor_network=get_feature_extractor("CNN"),
        recurrent_policy=True,
        restart_model=None)

    # 5. Define rollouts storage
    storage_factory = GAEBuffer.create_factory(size=4096, gae_lambda=0.95)

    # 6. Define scheme
    params = {}

    # add core modules
    params.update({
        "algo_factory": algo_factory,
        "actor_factory": actor_factory,
        "storage_factory": storage_factory,
        "train_envs_factory": train_envs_factory,
    })

    scheme = Scheme(**params)

    # 7. Define learner
    learner = Learner(scheme, target_steps=10e6, log_dir="./tmp/")

    # 8. Define train loop
    iterations = 0
    start_time = time.time()
    while not learner.done():

        learner.step()

        if iterations % 1 == 0:
            learner.print_info()

        # if iterations % 50 == 0:
            # save_name = learner.save_model()
            # args_dict.update({"latest_model": save_name})
            # args_path = os.path.join(args.log_dir, "training_arguments.json")
            # json.dump(args_dict, open(args_path, "w"), indent=4)

        # if args.max_time != -1 and (time.time() - start_time) > args.max_time:
        #     break

        iterations += 1

    print("Finished!")
    sys.exit()


if __name__ == "__main__":
    main()

```

**Comment by albertbou92** [comment]
Hi MarcoMeter,

In the current version, only algorithm-related information is displayed during training. We are working to allow specifying which information to print (algorithm information, performance information, debugging information, etc) in the next version via a parameter in the "learner.print_info()" method. Expect it in a few days.

However, the "performance/train" plot in Tensorboard does correspond to the training mean reward. It might look smoother than it should because right now we are using a running average with window length 100 during training to smooth information values. The same happens with loss curves, they have been smoothed previous to Tensorboard logging. We will remove this smoothing or at least allow adjusting window length value(with default value 1) in the next version since Tensorboard already allows to smooth curves.

Right now we check exact rewards from gym monitor.csv files stored in VecEnv log directory (in your case "/tmp/minigrid/train/"). The info_keywords parameter allows to include more information columns in the monitor.csv files specific to the environment (So it is correct, info_keywords are environment-specific).
